### DISCUSSION TITLE
Regarding version of JQuery

### DISCUSSION THREAD

**Discussion by Astronomia-maker** [ifm/ifm3d#442] [Q&A]
Hello support team, 

We did some penetration test on OVP800. For this we have given this task to external company and they have raised following issues.

- Upgrade to JQuery version 3.5.0 or later
- Return the X-Frame-Options or Content-Security-Policy (with the 'frame-ancestors' directive) HTTP header with the page's response.

Can you please clarify above question. Currently we are using firmware version 1.0.14. But for the penetration test we have used 0.14 or 0.16 (as i am not sure). 

So in latest firmware (1.0.xx) both issues are addressed or not?

**Comment by desengph** [comment]
@Astronomia-maker 
thank you for the information.
We will check the addressed questions above and let you know about the versions used etc.

**Comment by desengph** [comment]
@Astronomia-maker 

JQuery in the SWUpdate is updated to 3.6.0, otherwise we are not aware of any other usage of jQuery.
X-Frame-Options and Content-Security-Policy is not active.
Question: Is such handling requested for the recovery mode only, i.e. during the FW update process, or also when using any XML RPC configuration communications?
